### ISSUE TITLE
Bug Report: Cannot open legacy json files

### ISSUE THREAD

**Issue by An-m1654** [FunkinCrew/Funkin#3894]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Downloadable Build) - MacOS

### Browser

None

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

I tried dragging and dropping the json file into the editor, it didn't take my file, I tried clicking then selecting, but all the files are greyed (or whited) out, so I can't select then either.

### Steps to Reproduce

1. Open Charts Editor
2. press file
3. press new chart
4. press FNF Legacy
5. Cannot drop or select chart json file

**Comment by Hundrec** [comment]
I'm not sure I quite understand the context of this issue.
Can you provide a video of you attempting to open a .json?

**Comment by An-m1654** [comment]
https://github.com/user-attachments/assets/e8d6e6af-abcd-4763-b8b3-6ace08451c23

**Comment by JVNpixels** [comment]
That mod isn’t on the legacy version of FNF, that has new stuff, I don’t know why it wouldn’t let you select it though thats strange.

**Comment by NotHyper-474** [comment]
File drag-and-drop is disabled for MacOS but I'm puzzled on why it won't let you select though.

**Comment by Hundrec** [comment]
Thanks for the video!
Maybe @AbnormalPoof can troubleshoot this one

**Comment by AbnormalPoof** [comment]
Confirmed
![Screenshot 2024-11-26 at 3 31 37 PM](https://github.com/user-attachments/assets/0ce26517-5889-493a-b2f2-2c2c15e60b7b)

**Comment by An-m1654** [comment]
> That mod isn’t on the legacy version of FNF, that has new stuff, I don’t know why it wouldn’t let you select it though thats strange.

tl;dr for this comment: this is irrelevant to the post
I do actually know it's new, I was just trying to turn this mod from windows to mac, and just copying the assets folder didn't work at all, and crashes, so I was trying to fix the charts and I found this issue. So actually, the mod I installed is irrelevant

**Comment by JVNpixels** [comment]
> > That mod isn’t on the legacy version of FNF, that has new stuff, I don’t know why it wouldn’t let you select it though thats strange.
> 
> tl;dr for this comment: this is irrelevant to the post I do actually know it's new, I was just trying to turn this mod from windows to mac, and just copying the assets folder didn't work at all, and crashes, so I was trying to fix the charts and I found this issue. So actually, the mod I installed is irrelevant

im just saying because maybe that is the reason but alright ig?
